tests/integration/server/users_api_test.py

```diff
@@ -191,7 +193,14 @@ def test_user_delete_api():
     with requests.Session() as session:
         response = create_test_user(session, user)
         next_id = response.json()["data"]["user"]["id"]
+        role_payload = {
+            "name": "my_test_role",
+            "owners": next_id,
+            "administrators": next_id,
+            "description": "This is a test Role",
+        }
         conn = connect_to_db()
+        insert_role(role_payload)
```

Comment:
Use create_test_role() function here to create the role and it's relationships in the blockchain.

Reply:
Also will need an assertion to check that the user is currently an admin of the newly created role.